conf.py

```diff
@@ -97,3 +98,7 @@
 }
 
 todo_include_todos = True
+
+# sphinxext-opengraph config
+ogp_site_url = "https://devguide.python.org/"
+ogp_image = "https://devguide.python.org/_static/python-logo.svg"
```

Comment:
Should we add [more variables](https://github.com/wpilibsuite/sphinxext-opengraph#options)?

Reply:
Re: other variables:

I'll check `ogp_custom_meta_tags` for `<meta name="description">`.

https://github.com/python/docs-community/issues/65#issuecomment-1270446743

Reply:
> I'll check `ogp_custom_meta_tags` for `<meta name="description">`.

We can use `ogp_custom_meta_tags` to add it, but it would be hardcoded and we'd like to generate it from each page.

Reply:
> The recommended image size is 1200 x 630, should we use a Python logo or something else?

Since the image is rectangular (and somewhat big), it would be nice if it had both the Python logo and the text "Devguide".  If/when we do the same for `cpython` (or for the `peps` repo), we can have the same logo but different texts, so that the links are easily distinguishable.

This however would require creating new images, so if you don't want to do that, just a simple logo will do (we can always update the image later).

> I don't think we need any of the other variables.

ok

> We can use `ogp_custom_meta_tags` to add it, but it would be hardcoded and we'd like to generate it from each page.

Right, it would be better if the description was created dynamically.

Reply:
SVG replaced with a PNG of the Python logo + "Devguide".

![og-image](https://user-images.githubusercontent.com/1324225/194956249-69b078c7-29f0-457b-9c64-58cfa45c6763.png)

I wrote a script to create it (https://github.com/hugovk/pixel-tools/pull/15), so we can re-use that for other repos if we want.

I also squashed out the extra bytes from the PNG using `optipng -o7 -zm1-9 _static/og-image.png` (57,118 -> 9,573 bytes, 16.76% decrease).

When temporarily setting `ogp_image = "https://cpython-devguide--953.org.readthedocs.build/_static/og-image.png"` Facebook's sharing debugger shows the following OG metadata/preview: 

<img width="800" alt="image" src="https://user-images.githubusercontent.com/1324225/194959706-e84d81d0-2c35-4f6d-8d9a-f956443e8445.png">

https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fcpython-devguide--953.org.readthedocs.build%2F

Reply:
It's taking a while to get the font. It would be nice to get this merged to see how it works when live. Shall we merge this as-is, or with the image removed?

Reply:
Let's merge it.
I'll ping Deb again about the font and we can update that in a follow-up PR.

Reply:
(Continued in main thread: https://github.com/python/devguide/pull/953#issuecomment-1328700892)